bundles/framework/personaldata/PublishedMapsTab.js

```diff
@@ -275,7 +275,55 @@ Oskari.clazz.define('Oskari.mapframework.bundle.personaldata.PublishedMapsTab',
             }
             return gridModel;
         },
-
+        _isCurrentProjectionSupportedForView: function (data) {
+            var currentProjection = Oskari.getSandbox().getMap().getSrsName();
+            var viewProjection = data.state.mapfull.state.srs;
+            if ( viewProjection !== currentProjection ) {
+                return false;
+            }
+            return false;
```

Comment:
You could just return viewProjection === currentProjection and get rid of 3 lines of code :)